apps/alerting/historian/kinds/v0alpha1/notification.cue

```diff
@@ -0,0 +1,59 @@
+package v0alpha1
+
+#NotificationStatus: "firing" | "resolved" @cog(kind="enum",memberNames="Firing|Resolved")
+
+#NotificationOutcome: "success" | "error" @cog(kind="enum",memberNames="Success|Error")
+
+#NotificationQuery: {
+    // From is the starting timestamp for the query in RFC3339Nano format.
+    from?: int64
+    // To is the starting timestamp for the query in RFC3339Nano format.
+    to?: int64
+    // Limit is the maximum number of entries to return.
+    limit?: int64    
+    // Receiver optionally filters the entries by receiver title (contact point).
+    receiver?: string
+    // Status optionally filters the entries to only either firing or resolved.
+    status?: #NotificationStatus
+    // Outcome optionally filters the entries to only either successful or failed attempts.
+    outcome?: #NotificationOutcome
+    // RuleUID optionally filters the entries to a specific alert rule.
+    ruleUID?: string
+    // GroupLabels optionally filters the entries by matching group labels.
+    groupLabels: #Matchers
+}
+
+#NotificationQueryResult: {
+    entries: [...#NotificationEntry]
+}
+
+#NotificationEntry: {
+    // Timestamp is the time at which the notification attempt completed in RFC3339Nano format.
+    timestamp: int64
+    // Receiver is the receiver (contact point) title.
+    receiver: string
+    // Status indicates if the notification was for alerts firing or alerts resolving.
+    status: #NotificationStatus
```

Comment:
I think there could be a better doc for this field, which is kinda aggregated status, and it's always firing if there is at least one firing alertin in `alerts`